package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
       "dev": true,
       "requires": {
-        "@babel/highlight": "^7.0.0"
+        "@babel/highlight": "7.0.0"
```

Comment:
This package-lock diff is unusually large. Could you make sure you're on the latest npm, reset this file, and then run `npm install` again to see if that tightens it up?

Reply:
Yep, updating to the latest version of `npm` seems to have helped a lot. 😉